src/__tests__/meili.ts

```diff
@@ -73,56 +79,45 @@ test('reset-start', async () => {
   await clearAllIndexes()
 })
 
-test('create-index-with-name', async () => {
-  const index = {
-    name: 'ABABABABA',
-  }
-  await expect(meili.listIndexes()).resolves.toHaveLength(0)
-
-  await expect(meili.createIndex(index)).resolves.toHaveProperty(
-    'name',
-    index.name
-  )
-
-  await expect(meili.listIndexes()).resolves.toHaveLength(1)
-
-  await expect(meili.createIndex(index)).resolves.toHaveProperty(
-    'name',
-    index.name
-  )
-
-  await expect(meili.listIndexes()).resolves.toHaveLength(2)
-
-  await clearAllIndexes()
-})
-
 test('create-index-with-uid', async () => {
   const index = {
-    name: 'ABABABABA',
     uid: 'abababa',
   }
+  const indexIndentifier = {
+    uid: 'bebebebe',
```

Comment:
Could we please use more "professional" example? 😇 Like `index_uid` or just `uid`?

Reply:
Done